app/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -99,6 +99,7 @@ data class WireColorScheme(
     val classifiedBannerForegroundColor: Color,
     val unclassifiedBannerBackgroundColor: Color,
     val unclassifiedBannerForegroundColor: Color,
+    val mlsVerifiedIconColor: Color,
```

Comment:
Does it apply only to the MLS verification icon? 🤔

Reply:
removed this color, as i noticed it's better to use `Image` instead of `Icon` in my cases